src/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -421,10 +523,6 @@ void CodegenNeuronCppVisitor::print_mechanism_global_var_structure(bool print_in
         // TODO implement these when needed.
     }
 
-    if (info.point_process) {
```

Comment:
@1uc Any idea what might be missing here? I couldn't find anything related.
I tried the following mod file:
```
NEURON {
    POINT_PROCESS test
    RANGE x
    GLOBAL s
}
PARAMETER {
    s = 2
}
ASSIGNED {
    x
}
PROCEDURE a() {
    x = get_42()
}
FUNCTION b() {
    a()
}
FUNCTION get_42() {
    get_42 = 42
}
```
and I saw the same code being generated for `s`.

Reply:
not sure the context for this comment but I tried this PR/latest changes and I see:

```console
       /** all global variables */
       struct test_Store {
           double s{2};
       };

       /** connect global (scalar) variables to hoc -- */
       static DoubScal hoc_scalar_double[] = {
           {"s_test", &test_global.s},
           {nullptr, nullptr}
       };

```

So I see necessary snippets generated for GLOBAL variable.

I will mark this comment resolved.